fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -238,7 +237,7 @@ public void contextDestroyed(ServletContextEvent event) {
                 natsPublisher = null;
             }
 
-            if (graphTermServiceProvider != null) {
+            for (GraphTermServiceProvider graphTermServiceProvider : graphTermServiceProviders) {
                 graphTermServiceProvider.getGraph().close();
```

Comment:
what are the repurcussions of failing to close a graph?

Reply:
The close is used to release resources used by the JanusGraph driver. If there is an exception thrown from the close it will be logged.